src/lib/util.ts

```diff
@@ -129,6 +129,7 @@ const magicNumberToMimeType = new Map<RegExp, string>([
   [/^II*/, 'image/tiff'],
   [/^MM\x00*/, 'image/tiff'],
   [/^RIFF....WEBPVP8[LX ]/, 'image/webp'],
+  [/\xF4\xFF\x6F/, 'image/webp2'], // Not sure about this one tho lol
```

Comment:
It should probably at least start with `^` so that we don't scan the whole file.